src/relayserver/RelayServer.js

```diff
@@ -286,7 +288,11 @@ class RelayServer extends EventEmitter {
         console.error('web3 subscription:', error)
       }
     }).on('data', this._workerSemaphore.bind(this)).on('error', (e) => { console.error('worker:', e) })
-    setTimeout(() => { this.web3.eth.getBlockNumber().then(blockNumber => this._workerSemaphore.bind(this)({ number: blockNumber })) }, 1)
+    const handler = () => { this.web3.eth.getBlockNumber().then(blockNumber => this._workerSemaphore.bind(this)({ number: blockNumber })) }
+    if (this.devMode) {
```

Comment:
Every 8 seconds in devMode? Are you sure?

Reply:
otherwise it just spams everything with logs

Reply:
This is only a buidler solution, it won't work on any testnet as this doesn't fix the websocket provider issue that we have.

Reply:
of course it spams. but if you check and call _worker only if blocknumber changes, it won't spam at all...